grails-app/views/managementUnit/index.gsp

```diff
@@ -116,11 +132,15 @@
             $('#admin-tab').tab('show');
     });
 
+    // Star button click event
```

Comment:
Once the function is moved into the view model, this can be replaced with a click binding.

Reply:
This has now been committed.